project/Dependencies.scala

```diff
@@ -15,21 +15,22 @@ object Dependencies {
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
 
   val junitVersion = "4.12"
-  val slf4jVersion = "1.7.25"
+  val slf4jVersion = "1.7.28"
   // check agrona version when updating this
-  val aeronVersion = "1.19.1"
+  val aeronVersion = "1.21.2"
   // needs to be inline with the aeron version
-  val agronaVersion = "1.0.1"
+  val agronaVersion = "1.0.7"
   val nettyVersion = "3.10.6.Final"
   val jacksonVersion = "2.9.9"
   val jacksonDatabindVersion = "2.9.9.3"
+  val protobufJavaVersion = "3.9.1"
 
   val scala212Version = "2.12.9"
   val scala213Version = "2.13.0"
 
   val reactiveStreamsVersion = "1.0.3"
 
-  val sslConfigVersion = "0.3.8"
+  val sslConfigVersion = "0.4.0"
```

Comment:
Consider whether we want this given that it drops hostname verification, https://github.com/lightbend/ssl-config/pull/138 . This means we might have to do hostname verification on 'our side' or describe what a user would have to do in our migration guide.

Reply:
[CI](https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/17170/testReport/junit/akka.stream.io/TlsSpec/SslTls_must_verify_hostname/) was failed due to ssl, so i'll revert it once.